tests/metatype/ttypetraits.nim

```diff
@@ -92,7 +92,46 @@ block distinctBase:
         doAssert($distinctBase(typeof(b2)) == "string")
         doAssert($distinctBase(typeof(c2)) == "int")
 
+block lenTuple:
+  type
+    VectorElementType = SomeNumber | bool
+    Vec[N : static[int], T: VectorElementType] = object
+      arr*: array[N, T]
+    Vec4[T: VectorElementType] = Vec[4,T]
+    Vec4f = Vec4[float32]
```

Comment:
> I added those and a few others in #13423; that PR just fixes the bug in lenTuple but doesn't change its name
Ok I see you copied the types in and then don't test anything.

Reply:
i just added `doAssert not compiles(lenTuple(Vec4f))` ; I'm actually not sure why you added these in https://github.com/nim-lang/Nim/pull/13401#issuecomment-586891927 ; the remaining ones are tuples and I do test for these